Source: Santiago M. Mola <coldwind@gentoo.org>
Upstream: Sent.
Reason: Reference test results are outdated.(Gentoo bug #238818).
--- plotutils-2.5.1.orig/test/plot2ps.xout	2008-09-26 23:48:15.000000000 +0200
+++ plotutils-2.5.1/test/plot2ps.xout	2008-09-26 23:48:35.000000000 +0200
@@ -1,5 +1,5 @@
 %!PS-Adobe-3.0 EPSF-3.0
-%%Creator: GNU libplot drawing library 4.2
+%%Creator: GNU libplot drawing library 4.3
 %%Title: PostScript plot
 %%CreationDate: Fri Dec  9 13:09:44 2005
 %%DocumentData: Clean7Bit
diff -ur plotutils-2.5.1.orig/test/plot2svg.xout plotutils-2.5.1/test/plot2svg.xout
--- plotutils-2.5.1.orig/test/plot2svg.xout	2008-09-26 23:48:15.000000000 +0200
+++ plotutils-2.5.1/test/plot2svg.xout	2008-09-26 23:57:27.000000000 +0200
@@ -2,10 +2,10 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
 <svg version="1.1" baseProfile="full" id="body" width="8in" height="8in" viewBox="0 0 1 1" preserveAspectRatio="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ev="http://www.w3.org/2001/xml-events">
 <title>SVG drawing</title>
-<desc>This was produced by version 4.2 of GNU libplot, a free library for exporting 2-D vector graphics.</desc>
+<desc>This was produced by version 4.3 of GNU libplot, a free library for exporting 2-D vector graphics.</desc>
 <rect id="background" x="0" y="0" width="1" height="1" stroke="none" fill="white"/>
-<g id="content" transform="translate(0,1) scale(1,-1) scale(0.00024414) " xml:space="preserve" stroke="black" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10.433" stroke-dasharray="none" stroke-dashoffset="0" stroke-opacity="1" fill="none" fill-rule="even-odd" fill-opacity="1" font-style="normal" font-variant="normal" font-weight="normal" font-stretch="normal" font-size-adjust="none" letter-spacing="normal" word-spacing="normal" text-anchor="start">
-<text transform="translate(2048,3222.5) scale(1,-1) " font-family="Times-Roman,Times,serif" font-size="184.32px" text-anchor="middle" alignment-baseline="text-after-edge" stroke="none" fill="black" >A Sample Plot</text>
+<g id="content" transform="translate(0,1) scale(1,-1) scale(0.00024414) " xml:space="preserve" stroke="black" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10.433" stroke-dasharray="none" stroke-dashoffset="0" stroke-opacity="1" fill="none" fill-rule="evenodd" fill-opacity="1" font-style="normal" font-variant="normal" font-weight="normal" font-stretch="normal" font-size-adjust="none" letter-spacing="normal" word-spacing="normal" text-anchor="start">
+<text transform="translate(2048,3262.7) scale(1,-1) scale(9.216) " font-family="Times-Roman,Times,serif" font-size="20px" text-anchor="middle" stroke="none" fill="black" >A Sample Plot</text>
 <rect x="1024" y="1024" width="2048" height="2048" stroke-width="4.8188" />
 <polyline points="1001.7,946.04 1001.7,886.46 994.21,882.73 " stroke-width="5.2876" stroke-linecap="round" stroke-linejoin="round" />
 <polyline points="1005.4,942.31 1005.4,886.46 1016.6,879.01 " stroke-width="5.2876" stroke-linecap="round" stroke-linejoin="round" />
